docs/tutorial/using-checkbox/checkbox-commands.rst

```diff
@@ -0,0 +1,296 @@
+
+.. code-block:: none
+
+    checkbox.checkbox-cli list
+    service 'service object'
+      children
+        provider 'com.canonical.plainbox:manifest'
+          children
+            file '/snap/checkbox22/current/lib/python3.10/site-packages/plainbox/impl/providers/manifest/README.md'
+            file '/snap/checkbox22/current/lib/python3.10/site-packages/plainbox/impl/providers/manifest/bin/plainbox-manifest-collect'
+    (...)
+            job 'com.canonical.certification::tpm2.0_4.1.1/tpm2_verifysignature'
+            job 'com.canonical.certification::tpm2.0_4.1.1/pcr0_mismatch_check'
+            job 'com.canonical.certification::tpm2.0_4.1.1/context_gap_max_check'
+            file '/snap/checkbox22/current/providers/checkbox-provider-tpm2/units/tpm2_4.1.1.pxu'
+
+That's **a lot** of objects! Fortunately, it is possible to retrieve objects
+from a specific group. Let's see what test plans are available, for instance:
+
+.. code-block:: none
+
+    checkbox.checkbox-cli list "test plan"
+
+    test plan 'com.canonical.certification::6lowpan-automated'
+    test plan 'com.canonical.certification::acpi-automated'
+    test plan 'com.canonical.certification::audio-cert-full'
+    (...)
+    test plan 'com.canonical.certification::tpm-cert-focal-automated'
+    test plan 'com.canonical.certification::tpm-cert-automated'
+
+Combined with a ``grep`` command, it is possible to find all the test plans
+related to audio testing:
+
+.. code-block:: none
+
+    checkbox.checkbox-cli list "test plan" | grep audio
+
+    test plan 'com.canonical.certification::audio-cert-full'
+    test plan 'com.canonical.certification::audio-cert-manual'
+    (...)
+    test plan 'com.canonical.certification::after-suspend-audio-pa-manual'
+    test plan 'com.canonical.certification::after-suspend-audio-automated'
+
+Or let's list all the available jobs (test cases):
+
+.. code-block:: none
+
+    checkbox.checkbox-cli list all-jobs
+    
+    id: com.canonical.certification::6lowpan/kconfig
+    kernel config options for 6LoWPAN
+    id: com.canonical.certification::IEEE_80211
+    Creates resource info for wifi supported protocols/interfaces
+    id: com.canonical.certification::acpi/oem_osi
+    test ACPI OEM _OSI strings
+    (...)
+
+By default, the output for the ``all-jobs`` option is to list the job
+identifier followed by its summary (or ``<missing summary>`` if there is no
+summary). We can tailor the output using the ``--format`` parameter and all the
+fields available from the jobs. To see what fields are available, run:
+
+.. code-block:: none
+
+    checkbox.checkbox-cli list all-jobs --format ?
+    
+    Available fields are:
+    _description, _purpose, _siblings, _steps, _summary, _verification, after, category_id, command, depends, environ, estimated_duration, flags, full_id, id, imports, plugin, require, requires, template-engine, template-filter, template-resource, template-unit, unit, user
```

Comment:
As above, I think that someone might find this information useful and reading in this form is not very handy. We should probably consider a better output to this command as well

Reply:
Fixed in [20d9258](https://github.com/canonical/checkbox/pull/635/commits/20d92587d4c1323c44666a647f24b991ccabe62d)